ui/src/app/views/cohort-edit/component.spec.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  async as _async,
```

Comment:
nit: Why alias? Other tests in the codebase I've seen don't do this (though let me know if they are inconsistent)

Reply:
`async` is a language keyword. Browsers are not yet throwing SyntaxErrors (and maybe never will, knowing how important backwards compatibility is to browser vendors) but it still makes me queasy to shadow what should be reserved words.  I hope Angular changes the function name soon, maybe to `runAsync` or something.

The `_` prefix is a Python convention for circumventing keywords that I've just brought over without thinking too much.

Reply:
I mean I can undo it I'm not really committed to dying on this hill.

Reply:
Actually I realized we don't import async in other places because we use `fakeAsync()` instead which allows for slightly more serial test code. Could consider using that instead (shouldn't matter for your current test); otherwise I'm not really opposed to the alias.

Reply:
Not to reverse my position too radically, but upon further thought, the reserved phrase is `async function`, and `async` by itself isn't reserved, so... I'm just reverting all the places I've aliased `async` to `_async` and calling it done.